package.json

```diff
@@ -3,27 +3,69 @@
   "version": "3.1.1",
   "description": "A way to add webrtc troubleshooting to your app",
   "main": "src/index.js",
+  "config": {
+    "report": "report --reporter=lcov"
+  },
   "scripts": {
-    "browserify": "node_modules/.bin/browserify -t [ babelify --presets [ es2015 ] ] --s WebRTCTroubleshooter src/index.js -o dist/webrtc-troubleshooter.bundle.js",
-    "uglify": "node_modules/.bin/uglifyjs dist/webrtc-troubleshooter.bundle.js -o dist/webrtc-troubleshooter.bundle.js",
+    "browserify": "browserify -t [ babelify --presets [ es2015 ] ] --s WebRTCTroubleshooter src/index.js -o dist/webrtc-troubleshooter.bundle.js",
```

Comment:
not sure what you mean here @jensengar, inside the package.json you don't have to reference node_modules/.bin it does it for you
https://docs.npmjs.com/misc/scripts